embargoes.install

```diff
@@ -22,15 +24,14 @@ function embargoes_schema() {
         'not null' => TRUE,
       ],
       'action' => [
-        'type' => 'varchar',
```

Comment:
@qadan , I believe the reason why a string was used here was so that a username would persist even after the user is deleted from the Drupal instance.